.github/workflows/publish.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Setup paket
       run: dotnet tool install Paket
     - name: Dotnet Pack
-      run: dotnet pack -c release -p:PackageVersion=${GITHUB_REF##*/v}
+      run: dotnet pack -c release -p:PackageVersion=${GITHUB_REF##*/v} -p:PackagePrefix=${GITHUB_REF##*/v}
```

Comment:
Do we need to set the `FileVersion` as well for this to work?

Reply:
we might, how can I check this easily?

Reply:
now I set `-p:Version=${GITHUB_REF##*/v}` only, according to the article you sent

Reply:
it should set `Version=AssemblyVersion=FileVersion`

Reply:
fair point, changed it to file versiion